src/services/services.ts

```diff
@@ -200,16 +202,19 @@ namespace ts {
         public pos: number;
         public end: number;
         public flags: NodeFlags;
+        public modifierFlagsCache: ModifierFlags;
+        public transformFlags: TransformFlags;
         public parent: Node;
         public symbol!: Symbol;
         public jsDocComments?: JSDoc[];
-        public transformFlags!: TransformFlags;
 
         constructor(pos: number, end: number) {
             // Set properties in same order as NodeObject
             this.pos = pos;
             this.end = end;
             this.flags = NodeFlags.None;
+            this.modifierFlagsCache = ModifierFlags.None;
+            this.transformFlags = undefined!; // TODO: GH#18217

```

Comment:
The default should be `TransformFlags.None`, which is the same as in https://github.com/Microsoft/TypeScript/blob/master/src/compiler/utilities.ts#L6973

Reply:
I copy and pasted this from `NodeObject` in the same file. Shall I also update that constructor?